default.nix

```diff
@@ -85,6 +85,60 @@ let
       });
     };
   });
+  rawDockerImage = let
+    topologyFile = pkgs.writeText "topology.yaml" ''
+      wallet:
+        relays:
+          [
+            [
+              { host: nodes.awstest.iohkdev.io }
+            ]
+          ]
+        valency: 1
+        fallbacks: 7
+    '';
+    CLUSTER = "mainnet-1.0";
+    SYSTEM_START_TIME = 1504820421;
+    configFiles = pkgs.runCommand "cardano-config" {} ''
+      mkdir -pv $out
+      cd $out
+      cp -vi ${cardanoPkgs.cardano-sl.src + "/configuration.yaml"} configuration.yaml
+      cp -vi ${cardanoPkgs.cardano-sl.src + "/mainnet-genesis.json"} mainnet-genesis.json
```

Comment:
I'd be suprised this works, because that's the wrong genesis for the given key. Maybe we could just copy `*-genesis*.json`?